c/zosfile.c

```diff
@@ -770,10 +783,166 @@ int symbolicFileInfo(const char *filename, BPXYSTAT *stats, int *returnCode, int
   return returnValue;
 }
 
+int fileChangeOwner(const char *fileName, int *returnCode, int *reasonCode, 
+                    int usrId, int grpId) {
+  int nameLength = strlen(fileName);
+  int *reasonCodePtr;
+  int returnValue = 0;
+  *returnCode = *reasonCode = 0;
+
+#ifndef _LP64
+  reasonCodePtr = (int*) (0x80000000 | ((int)reasonCode));
+#else
+  reasonCodePtr = reasonCode;
+#endif
+
+  BPXLCO(&nameLength,
+         fileName,
+         usrId,
+         grpId,
+         &returnValue,
+         returnCode,
+         reasonCodePtr);
+
+  if (fileTrace) {
+    if (returnValue != 0) {
+#ifdef METTLE
+      printf("BPXLCO FAILED: returnValue: %d, returnCode: %d, reasonCode: 0x%08x\n",
+             returnValue, *returnCode, *reasonCode);
+#else
+      printf("BPXLCO FAILED: returnValue: %d, returnCode: %d, reasonCode: 0x%08x, strError: (%s)\n",
+             returnValue, *returnCode, *reasonCode, strerror(*returnCode));
+#endif
+    }
+    else {
+      printf("BPXLCO (%s) OK: returnValue: %d\n\n", fileName, returnValue);
+    }
+  }
+
+  if (returnValue != 0) {
+    returnValue = -1;
+  }
+  return returnValue;
+}
+
+int gidGetUserInfo(const char *userName,  UserInfo * info,
```

Comment:
It doesn't seem to belong in `zosfile`. It's used to get user info. Same for `gidGetGroupInfo`.

Reply:
zosaccounts.c now